src/context/VernaContextProvider.tsx

```diff
@@ -196,14 +169,14 @@ function VernaContextProvider({
         fullSchema,
         fullUiSchema,
         handleSubmit,
-        readOnly,
+        isEditor,
+        objectFieldTemplate: { ...objectFieldTemplate, ...defaultObjectFieldTemplate },
```

Comment:
You should revert the order otherwise default value will always override the provided one
```suggestion
        objectFieldTemplate: { ...defaultObjectFieldTemplate, ...objectFieldTemplate },
```

Reply:
Hmm indeed it may be better like that